Userland/Libraries/LibWeb/Bindings/MainThreadVM.cpp

```diff
@@ -147,8 +148,8 @@ ErrorOr<void> initialize_main_thread_vm(HTML::EventLoop::Type type)
         }
 
         // 3. Let settings object be the current settings object.
-        // 4. If script is not null, then set settings object to script's settings object.
-        auto& settings_object = script ? script->settings_object() : HTML::current_settings_object();
```

Comment:
Is there a tracking issue in whatwg/html or the shadow realm repo for updating the other specs to this new 'principal' verbiage? If not, this commit would be a good place to start.

Reply:
This update in verbiage is covered for the HTML spec in the merge request of https://github.com/whatwg/html/pull/9893

Some of the other specs also have updated verbiage, such as https://github.com/whatwg/webidl/pull/1437

Not all specs have an open merge request with an update (such as the XHR one https://github.com/whatwg/xhr/pulls) but I think that will come once the HTML spec change is merged.